src/Common/src/CoreLib/Interop/Windows/Kernel32/Interop.FormatMessage.cs

```diff
@@ -105,8 +107,5 @@ private static bool TryGetErrorMessage(IntPtr moduleHandle, int errorCode, Strin
         // 
```

Comment:
This comment should stay with the constants.

Reply:
> the buffer can only be freed using LocalFree and it is a private API on onecore.

This part of the comment is no longer valid. LocalFree API is available even in UAP.

Reply:
Ah... then there's no point in doing the whole growth allocation approach, either.  I can change it to start with the stack memory, and then if that's insufficient, ask FormatMessage to allocate and then I can free it.